ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/fees/EIP1559.java

```diff
@@ -76,6 +79,23 @@ public long getForkBlock() {
     return initialForkBlknum;
   }
 
+  public boolean isValid(final Transaction transaction, final LongSupplier blockNumberSupplier) {
```

Comment:
This feels like it should be in `MainnetTransactionValidator` instead, with two new constructor args.  If we get a third format we can go to an enum set or bitfield.  As part of `validate(final Transaction transaction)` we check the valid types against the booleans or enum set/bitfield.  Perhaps we need to define two forks for testing, `EIP1559` and `EIP1559Only`

Reply:
Done